index.html

```diff
@@ -209,9 +210,87 @@ <h2><span id="circle"></span> Skills</h2>
         <li>Gitlab</li>
         <li>Terminal</li>
       </ul>
-    
+
     </div>
   </section>
+  <section class="contact-me">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
+    <form method="POST">

```

Comment:
- [ ] Kindly include an action attribute that should be similar to `https://formspree.io/f/{form_id}` to the `form`. This will ensure that the form data is submitted to the formspree service.